packages/msat-bin/msat-bin.0.8.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "SAT solver binary based on the msat library"
+license: "Apache"
+maintainer: ["guillaume.bury@gmail.com" "simon.cruanes.2007@m4x.org"]
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" { >= "4.03" }
+  "dune" {build}
+  "msat" { = version }
+  "camlzip"
+]
```

Comment:
```suggestion
  "containers" {with-test}
]
```

It may also need a lower bound, please check. This is due to
```
#=== ERROR while compiling msat-bin.0.8.1 =====================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/Gbury/msat/archive/v0.8.1.tar.gz)
# path                 ~/.opam/4.09/.opam-switch/build/msat-bin.0.8.1
# command              ~/.opam/4.09/bin/dune runtest -p msat-bin -j 2
# exit-code            1
# env-file             ~/.opam/log/msat-bin-23-12e121.env
# output-file          ~/.opam/log/msat-bin-23-12e121.out
### output ###
# File "tests/dune", line 31, characters 0-165:
# 31 | (alias
# 32 |   (name runtest)
# 33 |   (package msat-bin)
# 34 |   (deps   ./../src/main/main.exe ./run (source_tree .))
# 35 |   (locks test)
# 36 |   (action (run /usr/bin/time -f "%e" ./run sat)))
# Error: File unavailable: /usr/bin/time
# File "src/main/dune", line 7, characters 13-23:
# 7 |   (libraries containers camlzip msat msat.sat msat.backend)
#                  ^^^^^^^^^^
# Error: Library "containers" not found.
# Hint: try: dune external-lib-deps --missing -p msat-bin -j 2 @runtest
```